elasticsearch_collectd.py

```diff
@@ -582,14 +581,20 @@
 
 
 # collectd callbacks
-def read_callback():
+def read_callback(cluster):
     """called by collectd to gather stats. It is called per collection
     interval.
     If this method throws, the plugin will be skipped for an increasing amount
     of time until it returns normally again"""
     log.info('Read callback called')
-    for c in CLUSTERS:
-        c.fetch_stats()
+
+    # determine node information
+    cluster.load_es_info()
```

Comment:
This will run every single read callback invocation right?  It should cache cluster name between reads to reduce load on the ES node.

Reply:
Yes, that's a good point. Putting it back where it originally was.